features/v1_features/latest_version_features/getting_a_job.feature

```diff
@@ -1,24 +1,30 @@
-Feature: Getting a job
+Feature: Getting a list of jobs
```

Comment:
The "Getting a list of jobs" feature already exists in the getting_jobs.feature so these scenarios should be added to that file instead.

Reply:
Done